src/spikeinterface/sorters/external/mountainsort5.py

```diff
@@ -169,13 +178,19 @@ def _run_from_folder(cls, sorter_output_folder, params, verbose):
             block_sorting_parameters=scheme2_sorting_parameters, block_duration_sec=p["scheme3_block_duration_sec"]
         )
 
-        scheme = p["scheme"]
-        if scheme == "1":
-            sorting = ms5.sorting_scheme1(recording=recording, sorting_parameters=scheme1_sorting_parameters)
-        elif p["scheme"] == "2":
-            sorting = ms5.sorting_scheme2(recording=recording, sorting_parameters=scheme2_sorting_parameters)
-        elif p["scheme"] == "3":
-            sorting = ms5.sorting_scheme3(recording=recording, sorting_parameters=scheme3_sorting_parameters)
+        with TemporaryDirectory() as tmpdir:
```

Comment:
also, why not just:
```suggestion
        with tempfile.TempraryDirectory() as tmpdir:
```
?

It seems that the MS5 wrapper is only adding context, which already exists for `tempfile.TempraryDirectory()`. Or am I missing something? (see [example](https://docs.python.org/3/library/tempfile.html#tempfile-examples))

Also, this will use `/tmp`, which is probably too small for general recordings...